### PR TITLE
Fix unstable test

### PR DESCRIPTION
**Pull request by hjaremko**

### CODE REVIEW

File: src/main/java/pl/uj/io/cuteanimals/model/Player.java [hunk 1]
```diff
@@ -15,18 +15,22 @@
  * @since 0.2.0-SNAPSHOT
  */
 public class Player implements IPlayer {
-    private final PlayerAttributes stats = new PlayerAttributes(this);
+    private final PlayerAttributes stats;
     private ILocation currentLocation = WorldMap.getInstance().getLocation("town");
     private final IEquipment armorBackpack = new ArmorBackpack(this);
     private final IEquipment backpack = new PlayerBackpack(this);
```
Comment (seqre) [inline src/main/java/pl/uj/io/cuteanimals/model/Player.java:21]: Please move to constructor.

File: src/test/java/pl/uj/io/cuteanimals/action/ability/FocusTest.java [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
 class FocusTest {
+    @Mock private RandomInteger random;
     private Player player;
     private Monster dummy;
     private PlayerAttributes attrs;
 
     @BeforeEach
     void setUp() {
-        player = new Player();
+        MockitoAnnotations.initMocks(this);
```
Comment (seqre) [inline src/test/java/pl/uj/io/cuteanimals/action/ability/FocusTest.java:25]: Remove and use `@ExtendWith(MockitoExtension.class)` before class declaration.

File: src/test/java/pl/uj/io/cuteanimals/model/PlayerAttributesTest.java [hunk 3]
```diff
@@ -1,13 +1,32 @@
...
+import org.mockito.MockitoAnnotations;
+import pl.uj.io.cuteanimals.model.interfaces.RandomInteger;
 
 class PlayerAttributesTest {
+    @Mock private RandomInteger random;
+    private PlayerAttributes attrs;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
```
Comment (seqre) [inline src/test/java/pl/uj/io/cuteanimals/model/PlayerAttributesTest.java:19]: Remove and use `@ExtendWith(MockitoExtension.class)` before class declaration.

File: src/main/java/pl/uj/io/cuteanimals/model/Player.java (hunk 1 above)
Comment (hjaremko) [reply src/main/java/pl/uj/io/cuteanimals/model/Player.java:21]: This will increase chance for error

**Pull request merged**